file.rb

```diff
@@ -0,0 +1,52 @@
+require 'json'
+
+module BookData
+  def read_book
+    if File.exist?('./files/books_data.json')
+      File.open('./files/books_data.json', 'r') do |file|
+        all_books = file.read
+        JSON.parse(all_books)
+      end
+    else
+      File.open('./files/books_data.json')
+    end
+  end
+
+  def write_book(newbook)
+    File.write('./files/books_data.json', JSON.pretty_generate(newbook))
+  end
+end
+
+module PeopleData
+  def read_people
+    if File.exist?('./files/peoples_data.json')
+      File.open('./files/peoples_data.json', 'r') do |file|
+        all_people = file.read
+        JSON.parse(all_people)
+      end
+    else
+      File.open('./files/peoples_data.json')
+    end
+  end
+
+  def write_people(newperson)
+    File.write('./files/peoples_data.json', JSON.pretty_generate(newperson))
+  end
+end
+
+module RentalData
+  def read_rental
+    if File.exist?('./files/rentals_data.json')
+      File.open('./files/rentals_data.json', 'r') do |file|
+        all_rentals = file.read
+        JSON.parse(all_rentals)
+      end
+    else
+      File.open('./files/rentals_data.json')
+    end
+  end
+
+  def write_rental(newrental)
+    File.write('./files/rentals_data.json', JSON.pretty_generate(newrental))
+  end
+end
```

Comment:
- [ ] @degisew and @Ceemos96 great work on your application, however, am getting the error below when one of the JSON files is deleted, kindly check on your implementation, the write method should be in the else statement instead of being on its own, the will ensure in case the file is missing you generate another one as per the project requirement
![image](https://user-images.githubusercontent.com/61408860/212461558-cb3ee867-5294-4f9c-8ef3-c5c72bb4983c.png)
![image](https://user-images.githubusercontent.com/61408860/212461624-45798fe2-c7c6-4f6c-ab3e-8416e2ce1c43.png)

Reply:
Okay, Thanks! We will fix it

Reply:
Great work making this change.